pikaday/build.boot

```diff
@@ -5,22 +5,21 @@
 
 (require '[cljsjs.boot-cljsjs.packaging :refer :all])
 
-(def +lib-version+ "1.5.1")
-(def +version+ (str +lib-version+ "-2"))
+(def +lib-version+ "1.8.0")
 
 (task-options!
   pom {:project 'cljsjs/pikaday
-       :version +version+
+       :version +lib-version+
```

Comment:
`+version+` var must not be removed. Just keep it like previous and reset build identifier (the part after dash) to zero.